src/components/interfaces/MOBILE_API.xml

```diff
@@ -3973,8 +3957,68 @@
         <param name="weatherServiceData" type="WeatherServiceData" mandatory="false"/>
     </struct>
 
+    <enum name="ServiceUpdateReason" since="5.1">
+        <element name = "PUBLISHED">
+            <description> The service has just been published with the module and once activated to the primary service of its type, it will be ready for possible consumption.</description>
+        </element>
+        <element name = "REMOVED">
+            <description> The service has just been unpublished with the module and is no longer accessible</description>
+        </element>
+        <element name = "ACTIVATED">
+            <description> The service is activated as the primary service of this type. All requests dealing with this service type will be handled by this service.</description>
+        </element>
+        <element name = "DEACTIVATED">
+            <description> The service has been deactivated as the primary service of its type</description>
+        </element>
+        <element name = "MANIFEST_UPDATE">
+            <description> The service has updated its manifest. This could imply updated capabilities</description>
+        </element>
+    </enum>
+
+    <struct name="AppServiceCapability" since="5.1">
+        <param name="updateReason" type="ServiceUpdateReason" mandatory="false">
+            <description> Only included in OnSystemCapabilityUpdated. Update reason for service record.</description>
+        </param>
+        <param name="updatedAppServiceRecord" type="AppServiceRecord" mandatory="true">
+            <description>Service record for a specific app service provider</description>
+        </param>
+    </struct>
+
+    <struct name="AppServicesCapabilities" since="5.1">
+        <description>Capabilities of app services including what service types are supported and the current state of services.</description>
+        <param name="servicesSupported" type="AppServiceType" array="true" mandatory="true">
```

Comment:
Change from `AppServiceType` to String

Reply:
This is handled in the base branch, changes should be merged